debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -710,6 +710,16 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                                       + "of the snapshot; in such cases set this property to 'false'.")
                                                               .withDefault(true);
 
+    public static final Field SNAPSHOT_PARALLEL = Field.create("snapshot.parallel")
+                                                       .withDisplayName("Snapshot newly added tables in parallel to normal binlog reading")
+                                                       .withType(Type.BOOLEAN)
+                                                       .withWidth(Width.SHORT)
+                                                       .withImportance(Importance.LOW)
+                                                       .withDescription("BETA FEATURE: If new tables are added to the config, snapshot the new tables in parallel to reading the binlog for the "
+                                                               + "tables previously in the config. Once reading of the new tables is caught up to the present time, both old and new readers will "
+                                                               + "be temporarily halted and a new binlog reader will start that will read the binlog for all configured tables.")
```

Comment:
Does this detect newly-added tables at any point, or are they detected only upon restart of the connector/task? Might be good to mention this.

Also, wdyt about "momentarily" rather than "temporarily". Maybe it's just me, but "momentarily" sounds shorter. :-)

Reply:
Fair enough, I'll clarify when new tables are detected. And I'll change the time wording :)